src/blind.lua

```diff
@@ -169,7 +173,60 @@ SMODS.Blind {
         max = 0
     },
     boss_colour = HEX('93a9ff'),
+    -- No base chips
     modify_hand = function(self, cards, poker_hands, text, mult, hand_chips)
         return mult, 0, hand_chips ~= 0
     end
 }
+
+SMODS.Blind {
+    key = "Samekh", -- The Ruler
+    atlas = "BakeryBlinds",
+    pos = {
+        y = 5
+    },
+    boss = {
+        min = 3,
+        max = 0
+    },
+    boss_colour = HEX('ffc201'),
+    --[[
+    artist = Jack5,
+    coder = Jack5,
+    ]]
```

Comment:
Why is this commented out?

Reply:
I'm pretty sure blinds don't support crediting, so the credits are in the code but go unused. If you can make them work, that would be great.

Reply:
Done in b7a38a2